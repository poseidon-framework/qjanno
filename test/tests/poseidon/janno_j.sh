qjanno "SELECT Poseidon_ID FROM j(data)"