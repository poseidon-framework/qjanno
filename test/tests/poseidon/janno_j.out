.-------------.
| Poseidon_ID |
:=============:
| XXX011      |
| XXX012      |
| XXX013      |
| XXX014      |
| XXX015      |
| XXX016      |
| XXX017      |
| XXX018      |
| XXX019      |
| XXX020      |
| XXX011      |
| XXX012      |
| XXX013      |
| XXX014      |
| XXX015      |
| XXX016      |
| XXX017      |
| XXX018      |
| XXX019      |
| XXX099      |
| XXX001      |
| XXX002      |
| XXX003      |
| XXX004      |
| XXX005      |
| XXX006      |
| XXX007      |
| XXX008      |
| XXX009      |
| XXX010      |
| XXX001      |
| XXX002      |
'-------------'
